app/src/main/java/com/example/android/hilt/di/DatabaseModule.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.android.hilt.di
+
+import android.content.Context
+import androidx.room.Room
+import com.example.android.hilt.data.AppDatabase
+import com.example.android.hilt.data.LogDao
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
+object DatabaseModule {
+    @Provides
+    @Singleton
+    fun provideDatabase(@ApplicationContext appContext: Context): AppDatabase {
+        return Room.databaseBuilder(
+            appContext,
+            AppDatabase::class.java,
+            "logging.db"
+        ).build()
+    }
+
+    @Provides
+    fun provideLogDao(database: AppDatabase): LogDao {
+        return database.logDao()
+    }
+}
```

Comment:
Don't forget to add the linter `ktlint` to Android Studio. It should be pointing out that you should add a new line